WBooks/Screens/BookLibrary/BookLibraryController.swift

```diff
@@ -22,6 +22,10 @@ class BookLibraryController: UIViewController {
         fatalError("init(nibName:bundle:) has not been implemented")
     }
     
+    lazy var libraryViewModel: BookLibraryViewModel = {
```

Comment:
I think it is not necessary to use a `lazy var`. `BookLibraryViewModel()` doesn't have any specific requirements  so it can be initialized as usual.